File: docs/manual/scala/guide/logging/code/docs/scaladsl/logging/LoggingExample.scala
/*
 * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
 */

package docs.scaladsl.logging

import org.slf4j.Logger
import org.slf4j.LoggerFactory

class LoggingExample {
  private final val log: Logger =
    LoggerFactory.getLogger(classOf[LoggingExample])

  def demonstrateLogging(msg: String): Unit = {
    log.debug("Here is a message at debug level: {}.", msg)
  }
}

File: docs/manual/java/guide/logging/code/docs/javadsl/logging/LoggingExample.java
/*
 * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
 */

package docs.javadsl.logging;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class LoggingExample {
  private final Logger log = LoggerFactory.getLogger(LoggingExample.class);

  public void demonstrateLogging(String msg) {
    log.debug("Here is a message at debug level: {}.", msg);
  }
}
